code/Equipment/Weapons/TeleportToolComponent.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Grubs.UI.Components;
+using System.Diagnostics;
+
+namespace Grubs.Equipment.Weapons;
+
+[Title( "Grub - Teleport Tool" ), Category( "Equipment" )]
```

Comment:
```suggestion
[Title( "Grubs - Teleport Tool" ), Category( "Equipment" )]
```

Reply:
Fixed in https://github.com/apetavern/grubs/pull/355/commits/23dbbb2ef5dd9dddee2079e4c739e9fe0dcb0a3c